React-Projects/test-react-app/src/UseEffectExample.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from "react";
+
+export default function UseEffectExample() {
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    document.title = `Count: ${count}`;
+  });
```

Comment:
This is not a right way to use useEffect from above code; it can be optimized. Please check all ways to use useEffect and then update the above code accordingly

Reply:
@abhi-codal  Understood